src/main/scala/com/github/tkawachi/doctest/CommentParser.scala

```diff
@@ -60,7 +60,8 @@ trait GenericParser extends RegexParsers {
     }
 
   private val verbatimBegin = {
-    val keywords = "abstract" | "case" | "class" | "def" | "implicit" | "import" |
+    val annotation = ("@" ~ anyStr1) ^^ append
+    val keywords = annotation | "abstract" | "case" | "class" | "def" | "implicit" | "import" |
```

Comment:
just a nitpick,
strictly speaking annotation is not a keyword, so instead of including it into keywords, on the last line you can say `annotation | keywords`

Reply:
You're right, dine.